.editorconfig

```diff
@@ -4,6 +4,6 @@ root = true
 
 [*]
 charset = utf-8
-indent_style = tab
+indent_style = space
```

Comment:
This is one is a bit tough. Tabs tend to get mixed with spaces in odd ways. Much of the JS tooling generates configuration files with spaces so we've gone with the path of least resistance.